app/app.go

```diff
@@ -78,9 +82,8 @@ var (
 		params.AppModuleBasic{},
 		// clp.AppModuleBasic{},
 		upgrade.AppModuleBasic{},
-		// oracle.AppModuleBasic{},
-		// ethbridge.AppModuleBasic{},
-		// faucet.AppModuleBasic{},
+		oracle.AppModuleBasic{},
```

Comment:
Please remove commented code

Reply:
Don't these need to be re-added once they are upgraded?